examples/delaney/delaney_weave.py

```diff
@@ -1,63 +0,0 @@
-"""
-Script that trains weave models on delaney dataset.
-"""
-from __future__ import print_function
-from __future__ import division
-from __future__ import unicode_literals
-
-import numpy as np
-np.random.seed(123)
-import tensorflow as tf
-tf.set_random_seed(123)
-import deepchem as dc
-
-# Load Delaney dataset
-delaney_tasks, delaney_datasets, transformers = dc.molnet.load_delaney(
-    featurizer='Weave', split='index')
-train_dataset, valid_dataset, test_dataset = delaney_datasets
-
-# Fit models
-metric = dc.metrics.Metric(dc.metrics.pearson_r2_score, np.mean)
-
-max_atoms_train = max([mol.get_num_atoms() for mol in train_dataset.X])
-max_atoms_valid = max([mol.get_num_atoms() for mol in valid_dataset.X])
-max_atoms_test = max([mol.get_num_atoms() for mol in test_dataset.X])
-max_atoms = max([max_atoms_train, max_atoms_valid, max_atoms_test])
-
-n_atom_feat = 75
-n_pair_feat = 14
-# Batch size of models
-batch_size = 64
-n_feat = 128
-graph = dc.nn.SequentialWeaveGraph(
-    max_atoms=max_atoms, n_atom_feat=n_atom_feat, n_pair_feat=n_pair_feat)
-
-graph.add(dc.nn.WeaveLayer(max_atoms, 75, 14))
-#graph.add(dc.nn.WeaveLayer(max_atoms, 50, 50))
-graph.add(dc.nn.WeaveConcat(batch_size, n_output=n_feat))
-graph.add(dc.nn.BatchNormalization(epsilon=1e-5, mode=1))
-graph.add(dc.nn.WeaveGather(batch_size, n_input=n_feat, gaussian_expand=True))
-
-model = dc.models.MultitaskGraphRegressor(
```

Comment:
I think there's a new Weave Tensorgraph version I believe.

Reply:
yeah it already exists in this folder so I didn't replace I just left the new one.